etc/testing/travis.sh

```diff
@@ -19,9 +25,54 @@ until kubectl version >/dev/null 2>/dev/null; do
 	sleep 1;
 done
 
+echo "Running test suite based on BUCKET=$BUCKET"
+
+PPS_SUITE=`echo $BUCKET | grep PPS > /dev/null; echo $?`
+
+make docker-build
+make clean-launch-dev
+make launch-dev
+
 go install ./src/server/cmd/match
-echo "Running local tests"
-make test
+
+if [[ "$BUCKET" == "MISC" ]]; then
+	echo "Running misc test suite"
+	make test-misc
+elif [[ $PPS_SUITE -eq 0 ]]; then
+	PART=`echo $BUCKET | grep -Po '\d+'`
+	NUM_BUCKETS=`cat etc/build/PPS_BUILD_BUCKET_COUNT`
+	echo "Running pps test suite, part $PART of $NUM_BUCKETS"
+	LIST=`go test -v  ./src/server/ -list ".*" | grep -v ok | grep -v Benchmark`
+	COUNT=`echo $LIST | tr " " "\n" | wc -l`
+	BUCKET_SIZE=$(( $COUNT / $NUM_BUCKETS ))
+	MIN=$(( $BUCKET_SIZE * $(( $PART - 1 )) ))
+	#The last bucket may have a few extra tests, to accommodate rounding errors from bucketing:
+	MAX=$COUNT 
+	if [[ $PART -ne $NUM_BUCKETS ]]; then
+		MAX=$(( $MIN + $BUCKET_SIZE ))
```

Comment:
Not sure if we want this to be our style, but in double parentheses (and double brackets, array indexes, and array offsets and lengths), I recently learned that you don't need dollar signs if the whole token is a variable name (e.g. `X` vs `${X}${Y}`)

Reply:
Yea this may be something we think about. And more generally ... maybe we want to start a bash style guide? I read something recently that stated something like 'when in doubt ... double quote in bash' and that will rarely be 'wrong'. Since none of us are bash masters, I kind of like keeping quotes in place, because it's more consistent (and there are cases where not having quotes are a gotcha) even if its a bit more verbose than needed in some cases.